packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -186,19 +202,28 @@ export const Tooltip = ({
         ref={elementRef}
         className={targetWrapperClassName}
         onMouseEnter={(evt: MouseEvent) => {
-          setIsHoveringTarget(true);
+          setTimeout(
+            () => setIsHoveringTarget(true),
+            delaySettings && delaySettings.show,
+          );
```

Comment:
Separating the delay props will help here. This will become:

```jsx
setTimeout(
  () => {},
  showDelay
)
```

Considering that we set the default value for the delays to 0 👌